timeless/templates/employees/create_edit.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block header %}
+  <h1>{% block title %}Employees management - {% if action == 'create' %} Create {% else %} Update {% endif %} {% endblock %}</h1>
+{% endblock %}
+
+{% block content %}
+  <form method="post">
```

Comment:
@VladimirRyzhkov pls look at this approach https://github.com/timelesslounge/timelessis/blob/master/timeless/templates/restaurants/tables/create_edit.html#L10

Reply:
@vladarefiev Vlad, I fixed.